examples/src/main/kotlin/example/AxiosSearch.kt

```diff
@@ -47,13 +58,13 @@ class AxiosSearch(props: AxiosProps) : RComponent<AxiosProps, AxiosState>(props)
             url = "http://ziptasticapi.com/" + zipCode
             timeout = 3000
         }
-        axios(config).then { response ->
+        axios(config).then { response: AxiosResponse ->
             setState {
                 zipResult = ZipResult(response.data.country, response.data.state, response.data.city)
             }
-        }.catch { error: dynamic ->
+        }.catch { error: AxiosError ->
             setState {
-                zipResult = ZipResult("", "", "Find error, please open your console to learn detail.")
+                zipResult = ZipResult("Find error:", error.message, ". Please open your console to learn detail")
```

Comment:
let's add an `errorMessage` field to state instead of misusing ZipResult data class